xml/dns_versionbind.xml

```diff
@@ -304,7 +304,7 @@
     <param pos="0" name="service.family" value="PowerDNS"/>
     <param pos="0" name="service.product" value="Recursor"/>
   </fingerprint>
-  <fingerprint pattern="^PowerDNS Authoritative Server (\d\.[\d.]++(?:-rc\d)?) \(\w+@[\w.]+ built [\d\s]+\w*@[\w.-]*\)$">
+  <fingerprint pattern="^PowerDNS Authoritative Server (\d\.[\d\.]+(?:-rc\d)?) \(\w+@[\w.]+ built [\d\s]+\w*@[\w.-]*\)$">
```

Comment:
This character is valid in the context of this character class group, similar to the one at the end of the pattern.  I've tested this both ways with an online regex tester for Golang but I haven't tested it natively with Golang. Unless you have a concern I'll revert this specific change.

Reply:
Oh, the "." in the end of the pattern has the same issue, it matches everything, not just period.

Reply:
It should not, see my comment just above your last comment.  I updated it to be more specific and to provide a link.  This probably didn't send an updated notice.

Reply:
Thank you, that was news to me.